pocs/observatory.py

```diff
@@ -316,36 +320,48 @@ def update_tracking(self):
             else:
                 ra_direction = 'east'
 
```

Comment:
Could you add some more explanation to the docstring, for those of us who are ignorant of mounts. For example, what does 50 milliseconds measure here? An angle? A time?

Reply:
Yes, will add.

Reply:
Done. I'm going to merge but let me know if docstring doesn't make sense. I'd like to test these additions on PAN001 tonight.